pkgs/tools/misc/gay/test-colorizes-hello-world.nix

```
@@ -0,0 +1,9 @@
+{ stdenv, gay }:
+stdenv.mkDerivation {
+  name = "gay-test-colors-hello-world";
+  meta.timeout = 10;
+  buildCommand = ''
+  echo Hello World | ${gay}/bin/gay | cat -v | grep -v "Hello World"
+  touch $out
```

```suggestion
    echo Hello World | ${gay}/bin/gay | cat -v | grep -v "Hello World"
    touch $out
```
To be honest, this test seems very flaky, we're checking that a string isn't there? What else should happen? I don't think it's needed.

---

That's a good point, I chose to err on the side of more tests than not. When I wrote this I was thinking if is colorized, then there will be various ansi escape codes in the middle of the text. I can take it out if you think that's a better option.